src/client/components/Footer.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function Footer() {
+    return (
```

Comment:
The formatting here is not consistent with the other files. I'm seeing 4 spaces for indentation while 2 spaces is more common for JS (and also what you use in the other files).